packages/stark-core/src/validation/validators/is-iban/is-iban.validator.fn.ts

```diff
@@ -12,9 +12,10 @@ export const starkIsIBANValidatorName = "starkIsIBAN";
  * @param iban - The IBAN number to validate
  */
 export function starkIsIBAN(iban: string): boolean {
-	if (typeof iban === "string") {
+	const electronicIban = electronicFormatIBAN(iban);
+	if (typeof electronicIban === "string") {
```

Comment:
@SuperITMan just a suggestion: a unit test for this would help us to understand the issue and also ensure we have no issues/regressions in the future 😉

Reply:
Hey Christopher! 
Actually, this change is due to a change in typings of "class-validator" library. 
You're right, we'll add a test for this in the next release 👍